content/md/en/docs/tutorials/connect-other-chains/local-relay.md

```diff
@@ -0,0 +1,178 @@
+---
+title: Prepare a local parachain testnet
+description:
+keywords:
+  - cumulus
+  - relay chain
+  - parachain
+  - parathread
+  - paraID
+  - rococo
+  - xcm
+  - xcmp
+  - collator
+  - testnet
+  - local
+---
+
+In this tutorial, you will configure a local relay chain and connect a parachain template for use in a local testing environment.
+
+## Before you begin
+
+Before you begin, verify the following:
+
+- You have configured your environment for Substrate development by installing [Rust and the Rust toolchain](/main-docs/install/).
+
+- You have completed [Build a local blockchain](/tutorials/get-started/build-local-blockchain/) tutorial previously.
+
+- You understand how to generated chain specifications for a private network of trusted validators as described in the [Add trusted nodes](/tutorials/get-started/trusted-network/) tutorial .
+
+- You are generally familiar with [architecture of Polkadot](https://wiki.polkadot.network/docs/learn-architecture) and [parachains](https://wiki.polkadot.network/docs/learn-parachains).
+
+## Tutorial objectives
+
+By completing this tutorial, you will accomplish the following objectives:
+
+- Identify software requirements.
+- Set up your parachain build environment.
+- Prepare a relay chain specification.
+- Start a relay chain locally.
+
+## Matching versions are critical
```

Comment:
I think you should remove this heading and move the content under Before you begin. That's the point of the section.

Reply:
I intend to reference this in other places, and the header provides a direct link. It also is of such importance I feel it motivates a section.